homeassistant/components/hive/climate.py

```diff
@@ -42,6 +57,19 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     add_entities([climate])
 
+    @refresh_system
+    def heating_boost(service):
+        """Handle the service call."""
+        node_id = HiveSession.entity_lookup[service.data.get(ATTR_ENTITY_ID)]
+        minutes = service.data.get(ATTR_TIME_PERIOD)
+        temperature = float(service.data.get(ATTR_TEMPERATURE, 25))
```

Comment:
Put the default in the service schema and coerce to float in the schema.

Reply:
I have put the default in the service schema but i am not sure how to represent a float with config validation. the only option i could find is small_float but that has a maximum of 1

Reply:
Use `vol.Coerce(float)`.